### PR TITLE
add fulcrum support

### PR DESCRIPTION
**Pull request by islandbitcoin**

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -84,8 +84,14 @@ dependencies:
   electrs:
     version: ">=0.9.6 <0.10.0"
     requirement:
-      type: "opt-out"
-      how: Set Enable Electrs to Disabled in the config
+      type: "opt-in"
```
Comment (chrisguida) [inline manifest.yaml:87]: Both are opt-in? Does that mean address indexing is off by default?

**Comment by chrisguida** [comment]
Needs a migration

**Comment by elvece** [comment]
hey @islandbitcoin is this still relevant?

**Comment by islandbitcoin** [comment]
> hey @islandbitcoin is this still relevant?

We didn't ever release fulcrum to the community, so i think this issue can be closed as not planned.

**Comment by chrisguida** [comment]
Fulcrum is a dumpster fire on the Pi but it might be worth releasing for other platforms

**Comment by elvece** [comment]
Noted, closing for now, can reopen when/if fulcrum is released

**Pull request closed without merging**